src/main/java/me/justeli/coins/handler/DropHandler.java

```diff
@@ -38,16 +39,20 @@
     private final Coins coins;
     private final NamespacedKey playerDamage;
 
+    private final LevelledMobsHandler levelledMobsHandler;
```

Comment:
i would prefer if the hook was initialized in the main class, just like with mythicmobs, and then you can move `getLevelledMobsMultipliedAmount()` to LevelledMobsHandler as well, and use the created `main.lmHook()` for `getLevelledMobsMultipliedAmount()`. 

for `deadMob`, idrk what's going on there, but i guess you could add an `@EventHandler` in the hook as well and register the hook (if lm is installed)

i just prefer to keep classes to their main usage, so if things like `deadMob` and `getLevelledMobsMultipliedAmount()` are from another plugin, i'd rather keep it in a separate class

Reply:
done